### ISSUE TITLE
Kubectl completion doesn't work on macOS

### ISSUE THREAD

**Issue by moicalcob** [CodelyTV/dotly#206]
The autocompletion for kubectl is not working on macOS. I've tried some different approaches to configure it, but it's not working 😞 

* Add the autocompletion script directly in .zshrc:
<img width="579" alt="Screenshot 2022-04-10 at 12 45 28" src="https://user-images.githubusercontent.com/56086628/162614437-78cf1bc9-c1bd-4512-b9c9-306ffa21e3f0.png">

* Create a file called `_kubectl` inside the completions folder in the .dotfiles folder with the autocompletion script content

**Comment by gtrabanco** [comment]
Have you tried to do a `dot zsh reload_completions` after adding?

**Comment by moicalcob** [comment]
Yes @gtrabanco! I did `dot shell zsh reload_completions` and it didn't work
